src/content/learn/add-react-to-an-existing-project.md

```diff
@@ -1,59 +1,59 @@
 ---
-title: Add React to an Existing Project
+title: Přidání Reactu do stávajícího projektu
 ---
 
 <Intro>
 
-If you want to add some interactivity to your existing project, you don't have to rewrite it in React. Add React to your existing stack, and render interactive React components anywhere.
+Pokud máte v plánu přidat interaktivitu do svého současného projektu, není nutné celý projekt přepracovat do Reactu. Můžete jednoduše začlenit React do své stávající technologické sady a umístit interaktivní React komponenty kamkoliv je potřeba.
 
 </Intro>
 
 <Note>
 
-**You need to install [Node.js](https://nodejs.org/en/) for local development.** Although you can [try React](/learn/installation#try-react) online or with a simple HTML page, realistically most JavaScript tooling you'll want to use for development requires Node.js.
+**Pro vývoj v lokálním prostředí je nutné mít nainstalovaný [Node.js](https://nodejs.org/en/).** I když máte možnost si [vyzkoušet React](/learn/installation#try-react) online nebo na jednoduché HTML stránce, realita je taková, že většina JavaScriptových nástrojů, které budete pravděpodobně chtít použít pro vývoj, vyžaduje Node.js.
 
 </Note>
 
-## Using React for an entire subroute of your existing website {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
+## Implementace Reactu do adresářové cesty vaší stávající webové stránky {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
```

Comment:
I think it is hard to understand. It should be clear that you can use React only on part of the website. I think the "adresářové cesty" is misleading.
```suggestion
## Implementace Reactu do adresářové cesty vaší stávající webové stránky {/*using-react-for-an-entire-subroute-of-your-existing-website*/}
```